README.md

```diff
@@ -15,19 +15,22 @@ ember install ember-cli-notifications
 
 ## Usage
 
-From within your controller or route.
-
 ### Add a notification
 
+There are four types of notification available.
```

Comment:
Hmm... this is a bit of a strange one really. We are not *changing* anything so this isn't really a major upgrade (i.e. all existing code will still work) but we are essentially saying "throw out the old way of doing it and use this new API" 

I kinda feel like we need a preamble that mentions this and maybe links to an older version of the docs to allow people to see what's changed.